services/pokemon_service.go

```diff
@@ -25,3 +42,18 @@ func FindPokemonById(id int) (*models.Pokemon, error) {
 	}
 	return nil, nil
 }
+
+func (pks PokemonService) LoadPokemons() (bool, error) {
```

Comment:
Do you actually need to return a boolean? I couldn't see it being used anywhere 🤔

Reply:
It would be nice to move the logic that takes care of communication with the external api to another package, like the repository one for example. Just like you have the logic to handle csv files there, you could do a very similar thing but with the logic that makes http requests and handles the responses.

This would mean that you will probably have two dependencies here, one that handles CSV file stuff, and another one for the http requests, and that's perfectly fine